src/ray/raylet/node_manager.cc

```diff
@@ -2202,11 +2203,16 @@ void NodeManager::PublishInfeasibleTaskError(const Task &task) const {
 }
 
 void NodeManager::SendSpilledObjectRestorationRequestToRemoteNode(
-    const ObjectID &object_id, const std::string &spilled_url, const NodeID &node_id) {
+    const ObjectID &object_id, const std::string &spilled_url, const NodeID &node_id,
+    std::function<void(const ray::Status &)> callback) {
   // Fetch from a remote node.
   if (!remote_node_manager_addresses_.contains(node_id)) {
```

Comment:
Can you make the address a required argument of this method? That way we can remove this entire error path and make it a problem for the pull manager.